app/2.0/docs/tools/polymer-cli-commands.md

```diff
@@ -0,0 +1,278 @@
+---
+title: Polymer CLI Commands
+---
+
+<!-- toc -->
+
+This section explains various useful Polymer CLI commands that you'll want to incorporate into your
+development workflow while you build your element or app project.
+
+The commands are intended for both element and app projects unless otherwise
+noted.
+
+* [`polymer build` (for app projects only)](#build)
+* [`polymer init`](#init)
+* [`polymer install`](#install)
+* [`polymer lint`](#lint)
+* [`polymer serve`](#serve)
+* [`polymer test`](#tests)
+* [Global options for Polymer CLI commands](#global)
+
+## `polymer build` {#build}
+
+*This command is for app projects only.*
+
+Generates a production-ready build of your app. This process includes minifying the HTML, CSS, and
+JS of the application dependencies, and generating a service worker to pre-cache dependencies.
+
+Polymer CLI's build process is designed for apps that follow the [app shell
```

Comment:
I think this link is a little confusing because some of our app structure is Polymer- or PRPL-specific (Like, the terms "entrypoint" and "fragment"). And that doc is full of a lot of implementation details that won't be helpful to the average Polymer user.

It would make more sense to link here to the PRPL doc, and have only that doc link to the app shell article for background on the app shell pattern.